filters/src/main/java/ua/net/tokar/json/rainbowrest/RainbowRestWebFilter.java

```diff
@@ -101,11 +107,14 @@ protected void doFilterInternal(
             }
         }
 
-        Set<String> include = new HashSet<>();
+        Map<String, List<NameValuePair>> include = new HashMap<>();
```

Comment:
I think a new structure should be introduces here, which will describe relationship between these parts of data.
for example:
```
class Part {
    string includeFieldName;
    list<Param> requestParams;
}

class Param {
    string key;
    string value;
}
```

Reply:
Why Part? maybe Include?

Reply:
as u wish :) it was for example